### PR TITLE
Makefile

### PR DESCRIPTION
**Pull request by rhatdan**
Adding the tests, I discovered that we were importing libcontainer into the selinux bindings.  In stead I copied the xattrs.go file into the selinux bindings.

Added Makefile to make test suite run.

### CODE REVIEW

File: selinux.go [hunk 1]
```diff
@@ -15,8 +15,6 @@ import (
 	"strings"
 	"sync"
 	"syscall"
```
Comment (mrunalp) [inline selinux.go]: Missing the xattr import in this file.

Comment (rhatdan) [inline selinux.go]: Fixed

**Comment by mrunalp** [comment]
LGTM

**Comment by runcom** [comment]
LGTM

**Comment by rhatdan** [comment]
So the test never ran, is there something we need to do to the repo to make the tests run?

**Comment by caniszczyk** [comment]
@rhatdan can you also accept your team invite to https://github.com/orgs/opencontainers/teams/go-selinux-maintainers ?

**Comment by rhatdan** [comment]
Link gives me a 404?

**Comment by caniszczyk** [comment]
@rhatdan check your email from an invite to GitHub, it says you haven't accepted the invite to join the team

**Comment by rhatdan** [comment]
@caniszczyk I can not find any email that mentions that, is there a link in github that I need to click?

**Comment by rhatdan** [comment]
@caniszczyk Ok I figured it out and "joined the team"

**Comment by mrunalp** [comment]
okay, the tests ran and passed! @rhatdan are you going to fix the labels to work on older policies in this PR or in a follow on?

**Comment by rhatdan** [comment]
I already did, I believe the following should work on older versions of selinux-policy.
```
+	if plabel != "user_u:user_r:user_t:s0:c1,c15" || (mlabel != "user_u:object_r:container_file_t:s0:c1,c15" && mlabel != "user_u:object_r:svirt_sandbox_file_t:s0:c1,c15") {
```

**Comment by mrunalp** [comment]
@rhatdan Thanks!
LGTM

**Pull request merged**